src/eliater/frontdoor_backdoor/example2.py

```diff
@@ -15,7 +15,7 @@
 from y0.graph import NxMixedGraph
 
 __all__ = [
-    "example_2",
+    "multiple_mediators_confounders_example",
```

Comment:
@pnavada please don't increase the diff by changing names of constants in this PR. I'm going to revert this again